elide-integration-tests/src/test/groovy/com/yahoo/elide/tests/FilterIT.groovy

```diff
@@ -867,6 +889,41 @@ class FilterIT extends AbstractIntegrationTestInitializer {
         }
     }
 
+    @Test
+    public void testGetBadRelationshipNameWithNestedFieldFilter() {
+        def result = mapper.readTree(RestAssured.get("book?filter[book.author12.name]=Null%20Ned").asString());
```

Comment:
I'm conflicted lol. Both *should* work. Should we have a case to verify it? Like a separate test case that uses `%20` for verification? Or do we not care that much? :)